src/incubator/Dialog/types.ts

```diff
@@ -69,15 +70,22 @@ export interface DialogHeaderProps extends ViewProps {
   onPress?: () => void;
 }
 
-export interface _DialogProps extends AlignmentModifiers, Pick<ViewProps, 'useSafeArea'> {
+export interface DialogCloseButton {
   /**
-   * The visibility of the dialog.
+   * Change the default close button text
    */
-  visible?: boolean;
+  closeText?: string;
```

Comment:
'label' is a better name IMO

Reply:
`closeLabel`?

Reply:
Just 'label' this is the type of the Dialog's 'closeButtonProps' prop (at the same time it can be ButtonProps type)...